llvm/docs/LangRef.rst

```diff
@@ -8642,8 +8642,11 @@ The '``llvm.used``' Global Variable
 The ``@llvm.used`` global is an array which has
 :ref:`appending linkage <linkage_appending>`. This array contains a list of
 pointers to named global variables, functions and aliases which may optionally
-have a pointer cast formed of bitcast or getelementptr. For example, a legal
-use of it is:
+have a pointer cast formed of bitcast or getelementptr. The address space of the
+pointers is always unspecified, rather than the globals address space, since
```

Comment:
Do not say unspecified. Say it is the default address space. 0 is not an unspecified address space